homeassistant/components/irish_rail_transport/sensor.py

```diff
@@ -194,9 +194,9 @@ def _empty_train_data(self):
                 ATTR_STATION: self.station,
                 ATTR_ORIGIN: "",
                 ATTR_DESTINATION: dest,
-                ATTR_DUE_IN: "n/a",
-                ATTR_DUE_AT: "n/a",
-                ATTR_EXPECT_AT: "n/a",
+                ATTR_DUE_IN: "0",
+                ATTR_DUE_AT: "0",
+                ATTR_EXPECT_AT: "0",
```

Comment:
The existing one and the new version offered in this PR are actually both incorrect. If a value isn't known, `None` must be returned.

../Frenck